homeassistant/components/flux_led/light.py

```diff
@@ -198,11 +198,18 @@ async def async_new_lights(bulbs: dict):
                 ),
             )
 
+            try:
+                host = bulb_details[CONF_HOST]
```

Comment:
Please only wrap the line that can raise in the try... except block.

Reply:
OK - done.